collated/main/A0139923X.md

```diff
@@ -0,0 +1,727 @@
+# A0139923X
+###### /java/seedu/todoList/commons/util/StringUtil.java
+``` java
+    public static boolean isUnsignedInteger(String s){
+        return s != null && s.matches("^\\d+$");
+    }
+    
+    /**
+     * Returns true if s represents an unsigned string e.g. A, AB, ABC, ... <br>
+     *   Will return false for null, empty string, numbers, " ABC " (untrimmed) "A B" (contains whitespace).
+     * @param s Should be trimmed.
+     */
+```
+###### /java/seedu/todoList/commons/util/StringUtil.java
+``` java
+    public static boolean isUnsignedString(String s){
+        return s != null && s.matches("^[a-zA-Z]+$");
+    }
+    
+```
+###### /java/seedu/todoList/commons/util/StringUtil.java
+``` java
+    public static boolean isUnsignedDate(String s){
+        return s != null && s.matches("^(\\d{2}-\\d{2}-\\d{4})$");
+    }
+}
+```
+###### /java/seedu/todoList/logic/parser/Parser.java
+``` java
+    private static final Pattern task_DATA_ARGS_FORMAT = // '/' forward slashes are reserved for delimiter prefixes
+            Pattern.compile("(?<name>.+)" 
+                    + " from/(?<date>[^/]+)" 
+                    + "( to/(?<endDate>[^/]+))?"
+                    + " p/(?<priority>[^/]+)");
+    
+```
+###### /java/seedu/todoList/logic/parser/Parser.java
+``` java
+    private static final Pattern event_DATA_ARGS_FORMAT = // '/' forward slashes are reserved for delimiter prefixes
+            Pattern.compile("(?<name>.+)" 
+                    + " from/(?<date>[^/]+)" 
+                    + "( to/(?<endDate>[^/]+))?"
+                    + " at/(?<startTime>[^/]+)"
+                    + " to/(?<endTime>[^/]+)");
+    
+```
+###### /java/seedu/todoList/logic/parser/Parser.java
+``` java
+    private static final Pattern deadline_DATA_ARGS_FORMAT = // '/' forward slashes are reserved for delimiter prefixes
+            Pattern.compile("(?<name>.+)" 
+                    + " on/(?<date>[^/]+)" 
+                    + " at/(?<endTime>[^/]+)");
+
+    /*
+     * Edit Command, task,event,deadline pattern
```

Comment:
Edit_**s**_ ...
please fix such similar violations in other header comments as well